private_gpt/server/chat/chat_service.py

```diff
@@ -97,8 +98,11 @@ def _chat_engine(
         system_prompt: str | None = None,
         use_context: bool = False,
         context_filter: ContextFilter | None = None,
+        context_template: str | None = None,
```

Comment:
Given we are not making use of this context_template argument anywhere, I'd suggest to remove it for the moment.